core/src/test/java/com/klaviyo/androidsdk/KlaviyoConfigTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.klaviyo.androidsdk
+
+import com.klaviyo.coresdk.KlaviyoConfig
+import org.junit.Test
+
+class KlaviyoConfigTest {
+    @Test
+    fun `Build Configuration works properly!`() {
```

Comment:
Can use this to specify any string we want as the function name for a unit test in Kotlin. Pretty cool but wonder what people think?

Reply:
love it, much easier to read test names, maybe we can decide on a naming scheme for tests?

Reply:
Seconded on Kenny's comment. Would also want to know

Reply:
Ok I'll think about this tomorrow and see if we can decide on a common test naming scheme!

Reply:
Nice, are we going to have a readme to document stylistic stuff and other things? or will there be a private wiki for that since this is public?

Reply:
Still thinking about that. I want to do that but I need to decide on the best place to put these things. I don't really want to use Github Wikis cause even though documenting a topic like this is harmless, there will be things to document like how we configure our locals down the road which we can't put there.

Reply:
I updated tests to follow a standard like this: `Unit Being Worked + State Under Test + Expected Behavior`. Which is pretty standard for unit tests, just a bit more vague to follow with human readable names.
So using these unit tests as an example it breaks down like this:
- **Unit Being Worked**: KlaviyoConfig Builder
- **State Under Test**: missing API key
- **Expected Behaviour**: throws expected exception